### DISCUSSION TITLE
Request Integration to Rabby wallet, support Darwinia/Crab chain by default

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#103] [Integration]
https://rabby.io/

Related #10

**Comment by nblogist** [comment]
Nice find @hackfisher Rabby looks rad!

**Reply by nblogist** [reply]
Reached out to their BD department, will keep this posted

**Comment by nblogist** [comment]
Just heard back from Eleanor at Debank, she asked to create a proposal at [Debank](https://debank.com/proposal), When doing so it asked to have a Web3 ID
![image](https://github.com/dcdao/discussion/assets/35498810/072d39c3-c29f-43b6-b2ed-6cbcb4be2f5d)

Which can be minted on the Debank platform 
![image](https://github.com/dcdao/discussion/assets/35498810/aa6523ee-efac-4f3f-bc54-2435520aa3f6)

But to generate one it costs $96 USDT/C on either of the following chain:
![image](https://github.com/dcdao/discussion/assets/35498810/26f57f19-2899-4be4-8e12-5106a0c92be6)

@hackfisher how can i request for the amount above? Or if someone already has a Web3ID minted, we can use that account to put up a proposal

**Reply by nblogist** [reply]
BUMP @hackfisher CC @1022ranji

**Reply by hackfisher** [reply]
> BUMP @hackfisher CC @1022ranji 

Please provide an address and request to @casinoyoda to send the funds required to you from the budget of Ecosystem program to create the id and proceed.

**Reply by nblogist** [reply]
Hi @casinoyoda here's an address: 0x822e4e429adedd91c96a81db39401a1770b96a5b
You can send on any of these chains 
![image](https://github.com/dcdao/discussion/assets/35498810/f8256e9c-ef1e-4743-a44f-6824a7209b1e)
And can send either USDT/USDC
I'd prefer polygon since it is cheaper.
Total amount requested is $96.
![image](https://github.com/dcdao/discussion/assets/35498810/d5970f6b-ee55-4f76-a0db-ad25dd6b6995)
Lmk if you can send the tokens directly will send RING, in which case please send $20 more to cover conversion, and other fees!
Thanks!

**Comment by casinoyoda** [comment]
I'll initiate a multisig transfer of some RING to you. Can it be done on Arbitrum? The transaction fees on Arb are also very cheap, and I currently don't have the transaction fees to execute it on Polygon. Right now, the price of RING on Arbitrum's Uniswap is 0.005542, so 96 USDT plus the Gas consumption is approximately 17666.5 RING. Do you need additional Gas for registering your Web3 ID?

**Reply by nblogist** [reply]
yes I'd need additional gas! I don't have anything on Arbitrum, also if you will send RING not converted, then lmk I'll send another address!

**Reply by casinoyoda** [reply]
If I transfer USDT to you on Arbitrum, how much do you need in total?

**Reply by nblogist** [reply]
$100 please 0x822E4e429aDeDd91c96a81db39401A1770B96a5b

**Reply by nblogist** [reply]
Can you also pls send some ARBs too :)

**Reply by casinoyoda** [reply]
Done~
0.001 arbETH https://arbiscan.io/tx/0xf762ca0cccd11448d8d5710d3b4228c39f58f41f39fa047cd675487951a1e10d
100 USDT https://arbiscan.io/tx/0x4a45e723c0360dc578008039c22114a20aba2278cdc813b9139f8dc49917d681

**Comment by nblogist** [comment]
Thanks! https://debank.com/proposal/65244 Added a proposal; Will send and email to Eleanor at Debank after we gather a good number of upvotes! Please share with community manager. :)
CC @hackfisher

**Comment by nblogist** [comment]
@hackfisher unfortunately this proposal got rejected, I wonder why tho, How do you want me to proceed? I can add the proposal again next month, perhaps it will have a better chance at passing, I think debank should provide a text body for these proposal to share why integration is important.
lmk @hackfisher !
![image](https://github.com/dcdao/discussion/assets/35498810/1adba161-b049-45e9-9259-d452f3da0bed)

**Reply by hackfisher** [reply]
It's ok, maybe just not enough votes, or small overlap of communities.

I think Rabby is a good wallet for multi chain and could be a good fit for cross-chain DApp using Darwinia Msgport.


Let's have a talk(one of the tri-weekly community calls in Asia time) and review on this first, and then have a look we can do.

**Comment by hackfisher** [comment]
https://github.com/RabbyHub/Rabby/issues/1274

**Reply by nblogist** [reply]
Damn that was a long thread for a future request I've seen; 
What I like about their Custom "Testnet" Feature is that you can quick add from Chainlist:

![image](https://github.com/dcdao/discussion/assets/35498810/edacab12-cbfa-4cee-b40f-2fb83abab689)

![image](https://github.com/dcdao/discussion/assets/35498810/3c7ca756-bf03-4743-8919-811f4128b514)
